fronts-client/src/actions/OptionsModal.ts

```diff
@@ -1,12 +1,13 @@
 import type { StartOptionsModal } from 'types/Action';
 import type { OptionsModalChoices } from 'types/Modals';
 import { ReactNode } from 'react';
+import { noop } from 'lodash';
```

Comment:
Do we need to bring in `'lodash'` just for this?

Reply:
We use lodash in many places, but I can see elsewhere in the project we (mostly) tend to use `lodash/${util}`, which probably does avoid some bundling weight – so I've replaced the few references I can find to destructuring imports in f093e3f767f0c413419068b94363f438ae7b84b1.